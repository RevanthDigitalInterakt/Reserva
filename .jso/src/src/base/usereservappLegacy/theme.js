  Object.defineProperty(exports, "__esModule", {
    value: true
  });
  exports.theme = undefined;
  var tokens = _interopRequireWildcard(_$$_REQUIRE(_dependencyMap[0]));
  function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(e) { return e ? t : r; })(e); }
  function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
  var theme = exports.theme = {
    space: {
      quarck: 4,
      nano: 8,
      micro: 12,
      xxxs: 16,
      xxs: 24,
      xs: 32,
      sm: 40,
      md: 48,
      xl: 64,
      xxl: 80
    },
    shadows: {
      bottomBarShadow: '0px -3px 2px rgba(0,0,0,0.1)',
      topBarShadow: '0px 3px 2px rgba(0,0,0,0.1)'
    },
    colors: {
      preto: tokens.preto,
      vermelhoFechadoRSV: tokens.vermelhoFechadoRSV,
      marinho: tokens.marinho,
      neutroFrio2: tokens.neutroFrio2,
      neutroQuente2: tokens.neutroQuente2,
      vermelhoRSV: tokens.vermelhoRSV,
      verde: tokens.verde,
      neutroQuente1: tokens.neutroQuente1,
      offWhite: tokens.offWhite,
      offWhite15: tokens.offWhite15,
      whiteSecondary: tokens.whiteSecondary,
      neutroFrio1: tokens.neutroFrio1,
      leve: tokens.leve,
      vermelhoAlerta: tokens.vermelhoAlerta,
      verdeSucesso: tokens.verdeSucesso,
      amareloAtencao: tokens.amareloAtencao,
      dropDownBorderColor: tokens.dropDownBorderColor,
      white: tokens.white,
      fullBlack: tokens.fullBlack,
      divider: tokens.divider,
      backgoundInput: tokens.backgoundInput,
      backgroundApp: tokens.backgroundApp,
      transparente: 'transparent',
      modalBackDropColor: tokens.modalBackDropColor,
      borderButton: tokens.borderButton,
      dividerCounter: tokens.dividerCounter,
      backgoundDivider: tokens.backgoundDivider,
      progressTextColor: tokens.progressTextColor,
      backgroundMenuOpened: tokens.backgroundMenuOpened,
      searchBarTextColor: tokens.searchBarTextColor,
      borderColorProduct: tokens.borderColorProduct
    },
    fonts: {
      nunitoBlack: 'Nunito-Black',
      nunitoBlackItalic: 'Nunito-BlackItalic',
      nunitoBold: 'Nunito-Bold',
      nunitoBoldItalic: 'Nunito-BoldItalic',
      nunitoExtraBold: 'Nunito-ExtraBold',
      nunitoExtraBoldItalic: 'Nunito-ExtraBoldItalic',
      nunitoExtraLight: 'Nunito-ExtraLight',
      nunitoExtraLightItalic: 'Nunito-ExtraLightItalic',
      nunitoItalic: 'Nunito-Italic',
      nunitoLight: 'Nunito-Light',
      nunitoLightItalic: 'Nunito-LightItalic',
      nunitoRegular: 'Nunito-Regular',
      nunitoSemiBold: 'Nunito-SemiBold',
      nunitoSemiBoldItalic: 'Nunito-SemiBoldItalic',
      reservaDisplayRegular: 'ReservaDisplay-Regular',
      reservaSerifBlack: 'ReservaSerif-Black',
      reservaSerifBlackItalic: 'ReservaSerif-BlackItalic',
      reservaSerifBold: 'ReservaSerif-Bold',
      reservaSerifBoldItalic: 'ReservaSerif-BoldItalic',
      reservaSerifItalic: 'ReservaSerif-Italic',
      reservaSerifLight: 'ReservaSerif-Light',
      reservaSerifLightItalic: 'ReservaSerif-LightItalic',
      reservaSerifMedium: 'ReservaSerif-Medium',
      reservaSerifMediumItalic: 'ReservaSerif-MediumItalic',
      reservaSerifRegular: 'ReservaSerif-Regular',
      reservaSerifThin: 'ReservaSerif-Thin',
      reservaSerifThinItalic: 'ReservaSerif-ThinItalic',
      reservaSansBlack: 'ReservaSans-Black',
      reservaSansBlackItalic: 'ReservaSans-BlackItalic',
      reservaSansBold: 'ReservaSans-Bold',
      reservaSansBoldItalic: 'ReservaSans-BoldItalic',
      reservaSansItalic: 'ReservaSans-Italic',
      reservaSansLight: 'ReservaSans-Light',
      reservaSansLightItalic: 'ReservaSans-LightItalic',
      reservaSansMedium: 'ReservaSans-Medium',
      reservaSansMediumItalic: 'ReservaSans-MediumItalic',
      reservaSansRegular: 'ReservaSans-Regular',
      reservaSansThin: 'ReservaSans-Thin',
      reservaSansThinItalic: 'ReservaSans-ThinItalic'
    },
    fontWeights: {
      light: 300,
      normal: 400,
      medium: 500,
      SemiBold: 600,
      bold: 700,
      black: 900
    },
    fontSizes: [4, 8, 12, 14, 16, 18, 20, 24, 27, 32, 48],
    radii: {
      pico: 2,
      nano: 5,
      xxxs: 10,
      xs: 15,
      sm: 20,
      pill: 500,
      infinity: 999999
    },
    borderWidths: {
      none: tokens.borderWidthNone,
      hairline: tokens.borderWidthHairline,
      thin: tokens.borderWidthThin,
      thick: tokens.borderWidthThick,
      heavy: tokens.borderWidthHeavy,
      borderDiscountTag: tokens.borderDiscountTag,
      stepper: 3
    }
  };
