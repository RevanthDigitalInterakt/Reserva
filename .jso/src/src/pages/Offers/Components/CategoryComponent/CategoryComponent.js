  Object.defineProperty(exports, "__esModule", {
    value: true
  });
  exports.default = CategoryComponent;
  var _react = _$$_REQUIRE(_dependencyMap[0])(_$$_REQUIRE(_dependencyMap[1]));
  var _reactNative = _$$_REQUIRE(_dependencyMap[2]);
  var _CardWrapper = _$$_REQUIRE(_dependencyMap[0])(_$$_REQUIRE(_dependencyMap[3]));
  var _testProps = _$$_REQUIRE(_dependencyMap[0])(_$$_REQUIRE(_dependencyMap[4]));
  function CategoryComponent() {
    var _useHomeStore = (0, _$$_REQUIRE(_dependencyMap[5]).useHomeStore)(['offersCarousels']),
      offersCarousels = _useHomeStore.offersCarousels;
    var title = offersCarousels.map(function (item) {
      var _item$categoryCards;
      return (_item$categoryCards = item.categoryCards) == null ? undefined : _item$categoryCards.sectionCardTitle;
    });
    return /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[6]).jsx)(_reactNative.View, Object.assign({}, (0, _testProps.default)('category_main_component'), {
      style: _$$_REQUIRE(_dependencyMap[7]).styles.mainContainer,
      children: /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[6]).jsxs)(_reactNative.View, {
        style: _$$_REQUIRE(_dependencyMap[7]).styles.childrenContainer,
        children: [/*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[6]).jsx)(_reactNative.View, {
          style: _$$_REQUIRE(_dependencyMap[7]).styles.containerRow,
          children: /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[6]).jsx)(_reactNative.Text, Object.assign({}, (0, _testProps.default)('category_main_title'), {
            style: _$$_REQUIRE(_dependencyMap[7]).styles.txtTitle,
            children: title
          }))
        }), /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[6]).jsx)(_CardWrapper.default, {})]
      })
    }));
  }
