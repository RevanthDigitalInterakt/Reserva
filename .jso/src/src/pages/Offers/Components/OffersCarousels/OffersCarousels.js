  Object.defineProperty(exports, "__esModule", {
    value: true
  });
  exports.OffersCarousels = OffersCarousels;
  var _defineProperty2 = _$$_REQUIRE(_dependencyMap[0])(_$$_REQUIRE(_dependencyMap[1]));
  var _react = _interopRequireWildcard(_$$_REQUIRE(_dependencyMap[2]));
  var _reactNative = _$$_REQUIRE(_dependencyMap[3]);
  var _testProps = _$$_REQUIRE(_dependencyMap[0])(_$$_REQUIRE(_dependencyMap[4]));
  var _OffersMainCarousel = _$$_REQUIRE(_dependencyMap[0])(_$$_REQUIRE(_dependencyMap[5]));
  function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(e) { return e ? t : r; })(e); }
  function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
  function OffersCarousels() {
    var _useHomeStore = (0, _$$_REQUIRE(_dependencyMap[6]).useHomeStore)(['offersCarousels', 'loading']),
      offersCarousels = _useHomeStore.offersCarousels,
      loading = _useHomeStore.loading;
    var renderCarousel = (0, _react.useCallback)(function (item) {
      if (!item.items.length) return null;
      var relationalObject = (0, _defineProperty2.default)((0, _defineProperty2.default)({}, _$$_REQUIRE(_dependencyMap[7]).HomePageSectionTypeEnum.Main, function () {
        return /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[8]).jsx)(_OffersMainCarousel.default, {
          data: item
        });
      }), "DEFAULT", function DEFAULT() {
        return null;
      });
      return (relationalObject[item.type] || relationalObject.DEFAULT)();
    }, []);
    if (loading) {
      return /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[8]).jsxs)(_$$_REQUIRE(_dependencyMap[9]).Box, {
        bg: "white",
        marginY: "nano",
        justifyContent: "center",
        children: [/*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[8]).jsx)(_reactNative.View, {
          style: {
            height: 100
          }
        }), /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[8]).jsx)(_reactNative.ActivityIndicator, {
          size: "small",
          color: _$$_REQUIRE(_dependencyMap[10]).COLORS.BLACK
        })]
      });
    }
    return /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[8]).jsx)(_reactNative.View, {
      style: {
        height: 400
      },
      children: offersCarousels.map(function (carousel) {
        return /*#__PURE__*/(0, _react.createElement)(_reactNative.View, Object.assign({}, (0, _testProps.default)('carousels_cards'), {
          key: `offers-item-${carousel.type}-${carousel.showtime}-${carousel.items.length}`
        }), renderCarousel(carousel));
      })
    });
  }
