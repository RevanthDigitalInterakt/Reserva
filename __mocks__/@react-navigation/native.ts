const navigate = jest.fn();

const useNavigation = () => ({
  navigate,
});

export {
  useNavigation,
};
