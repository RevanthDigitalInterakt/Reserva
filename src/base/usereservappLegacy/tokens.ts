export const white = '#FFF';
export const fullBlack = '#000';

// cores primarias
export const preto = '#333333';
export const vermelhoFechadoRSV = '#5A0000';
export const marinho = '#13294B';
export const neutroFrio2 = '#8A8C8E';
export const neutroQuente2 = '#B1ACA8';
export const divider = '#D8D9DA';
export const dividerCounter = '#EAEAEA';
export const progressTextColor = '#979797';
export const searchBarTextColor = '#666666';

// cores secundarias
export const vermelhoRSV = '#A00000';
export const verde = '#22372B';
export const neutroQuente1 = '#D7CFCA';
export const offWhite = '#D6D1C4';
export const offWhite15 = '#F9F8F6';
export const whiteSecondary = '#FAFAFA';
export const neutroFrio1 = '#D8D9DA';
export const leve = '#F9E1DD';

// cores do sistema
export const vermelhoAlerta = '#EF1E1E';
export const verdeSucesso = '#38A238';
export const amareloAtencao = '#FEA528';
export const dropDownBorderColor = '#DDDDDD';

// larguras de borda
export const borderWidthNone = 0;
export const borderWidthHairline = 1;
export const borderWidthThin = 2;
export const borderWidthThick = 4;
export const borderWidthHeavy = 8;
export const borderDiscountTag = 12;

// cores de borda
export const borderButton = '#333333';
export const borderColorProduct = '#757575';

// backgrounds
export const backgroundMenuOpened = '#F1F1F1';
export const backgroundApp = '#F9F9F9';
export const modalBackDropColor = '#AFAFAF';
export const backgoundInput = '#F0F0F0';
export const backgoundDivider = '#F5F5F5';
