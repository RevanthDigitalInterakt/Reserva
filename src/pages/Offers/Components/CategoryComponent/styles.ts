import { StyleSheet } from 'react-native';
import { COLORS, FONTS } from '../../../../base/styles';
import { scale } from '../../../../utils/scale';

export const styles = StyleSheet.create({
  mainContainer: {
    backgroundColor: COLORS.WHITE,
    flex: 1,
    paddingTop: scale(14),
    marginBottom: scale(24),
  },
  childrenContainer: {
    backgroundColor: COLORS.WHITE,
    paddingHorizontal: scale(14),
  },
  containerRow: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    alignItems: 'center',
  },
  txtTitle: {
    fontFamily: FONTS.RESERVA_SERIF_BOLD,
    fontSize: scale(18),
    lineHeight: scale(22),
    letterSpacing: -0.5,
    color: COLORS.LOADING_BORDER,
  },
  containerSeeAll: {
    flexDirection: 'row',
    gap: 2,
    alignItems: 'center',
  },
  txtSeeAll: {
    fontFamily: FONTS.NUNITO_REGULAR,
    fontSize: scale(12),
    lineHeight: scale(14),
    color: COLORS.GRAY_A6,
  },

});
