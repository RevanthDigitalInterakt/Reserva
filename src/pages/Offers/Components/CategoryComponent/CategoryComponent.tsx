import React from 'react';
import {
  Text,
  View,
} from 'react-native';
import { styles } from './styles';
import CardWrapper from './components/CardWrapper';
import { useHomeStore } from '../../../../zustand/useHomeStore';
import testProps from '../../../../utils/testProps';

export default function CategoryComponent() {
  const { offersCarousels } = useHomeStore(['offersCarousels']);

  const title = offersCarousels.map((item) => item.categoryCards?.sectionCardTitle);
  return (
    <View {...testProps('category_main_component')} style={styles.mainContainer}>
      <View style={styles.childrenContainer}>
        <View style={styles.containerRow}>
          <Text {...testProps('category_main_title')} style={styles.txtTitle}>{title}</Text>
        </View>
        <CardWrapper />
      </View>
    </View>
  );
}
