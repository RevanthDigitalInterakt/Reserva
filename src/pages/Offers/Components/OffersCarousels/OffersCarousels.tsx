import React, { useCallback } from 'react';
import { ActivityIndicator, View } from 'react-native';

import { useHomeStore } from '../../../../zustand/useHomeStore';
import { COLORS } from '../../../../base/styles/colors';
import type { OffersCarouselsOutput } from '../../../../base/graphql/generated';
import { HomePageSectionTypeEnum } from '../../../../base/graphql/generated';
import testProps from '../../../../utils/testProps';
import { Box } from '../../../../components/Box/Box';
import OffersMainCarousel from '../OffersMainCarousel/OffersMainCarousel';

export function OffersCarousels() {
  const { offersCarousels, loading } = useHomeStore(['offersCarousels', 'loading']);

  const renderCarousel = useCallback((item: OffersCarouselsOutput) => {
    if (!item.items.length) return null;

    const relationalObject: {
      [key in HomePageSectionTypeEnum | 'DEFAULT']: () => JSX.Element | null;
    } = {
      [HomePageSectionTypeEnum.Main]: () => (
        <OffersMainCarousel data={item} />
      ),
      DEFAULT: () => null,
    };

    return (relationalObject[item.type] || relationalObject.DEFAULT)();
  }, []);

  if (loading) {
    return (
      <Box bg="white" marginY="nano" justifyContent="center">
        <View style={{ height: 100 }} />

        <ActivityIndicator size="small" color={COLORS.BLACK} />
      </Box>
    );
  }

  return (
    <View style={{ height: 400 }}>
      {offersCarousels.map((carousel) => (
        <View
          {...testProps('carousels_cards')}
          key={`item-${carousel.type}-${carousel.showtime}-${carousel.items.length}`}
        >
          {renderCarousel(carousel)}
        </View>
      ))}
    </View>
  );
}
