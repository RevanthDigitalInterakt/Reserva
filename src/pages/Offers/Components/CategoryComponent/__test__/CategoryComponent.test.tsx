import React from 'react';
import { ThemeProvider } from 'styled-components/native';
import { MockedProvider } from '@apollo/client/testing';
import { render, screen } from '@testing-library/react-native';
import { theme } from '../../../../../base/usereservappLegacy/theme';
import CategoryComponent from '../CategoryComponent';
import { mockOffersCarouselsQuery } from './OffersCarousels.mock';

const mockNavigate = jest.fn();

jest.mock('@react-navigation/native', () => ({
  useNavigation: () => ({ navigate: mockNavigate }),
}));

jest.mock('../../../../../zustand/useHomeStore', () => ({
  useHomeStore: () => ({
    onLoad: () => {},
    offersCarousels: mockOffersCarouselsQuery.result.data.offersCarousels,
  }),
}));

const Component = (
  <ThemeProvider theme={theme}>
    <MockedProvider mocks={[mockOffersCarouselsQuery]} addTypename={false}>
      <CategoryComponent />
    </MockedProvider>
  </ThemeProvider>
);
describe('CategoryComponent', () => {
  beforeEach(async () => {
    render(Component);
  });
  it('should match to snapshot', () => {
    expect(screen.toJSON()).toMatchSnapshot();
  });

  it('should render items on screen', () => {
    const main = screen.getByTestId(
      'com.usereserva:id/category_main_component',
    );
    expect(main).toBeOnTheScreen();
  });
});
