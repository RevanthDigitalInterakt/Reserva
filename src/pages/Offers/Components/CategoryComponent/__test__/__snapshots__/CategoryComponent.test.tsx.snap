// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategoryComponent should match to snapshot 1`] = `
<View
  accessibilityLabel="com.usereserva:id/category_main_component"
  style={
    {
      "backgroundColor": "#ffffff",
      "flex": 1,
      "marginBottom": 56.25,
      "paddingTop": 32.8125,
    }
  }
  testID="com.usereserva:id/category_main_component"
>
  <View
    style={
      {
        "backgroundColor": "#ffffff",
        "height": 300,
        "paddingHorizontal": 32.8125,
      }
    }
  >
    <View
      style={
        {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "space-between",
        }
      }
    >
      <Text
        accessibilityLabel="com.usereserva:id/category_main_title"
        style={
          {
            "color": "#333333",
            "fontFamily": "ReservaSerif-Bold",
            "fontSize": 42.1875,
            "letterSpacing": -0.5,
            "lineHeight": 51.5625,
          }
        }
        testID="com.usereserva:id/category_main_title"
      >
        Navegue por categorias
      </Text>
    </View>
    <View
      accessibilityLabel="com.usereserva:id/category_wrapper_component"
      style={
        {
          "backgroundColor": "#ffffff",
          "flexDirection": "row",
          "flexWrap": "wrap",
          "gap": 28.125,
          "height": 190,
          "marginTop": 23.4375,
        }
      }
      testID="com.usereserva:id/category_wrapper_component"
    >
      <View
        accessibilityLabel="com.usereserva:id/category_card_item"
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "backgroundColor": "#ffffff",
            "borderColor": "#F8F8F8",
            "borderRadius": 8,
            "borderWidth": 1,
            "elevation": 0,
            "height": 150,
            "opacity": 1,
            "shadowColor": "#000000",
            "shadowOffset": {
              "height": 7,
              "width": 0,
            },
            "shadowOpacity": 0.6,
            "shadowRadius": 7,
            "width": 150,
          }
        }
        testID="com.usereserva:id/category_card_item"
      >
        <Image
          resizeMode="contain"
          source={
            {
              "uri": "https://images.ctfassets.net/6jsfqc13oxv4/3XYtC2MMnxjYGZCevi3WIy/dd489403865c05fbcf8b9a6f154374db/Screenshot_from_2024-08-15_18-23-14.png",
            }
          }
          style={
            [
              {
                "height": 562.5,
                "width": 750,
              },
              {
                "height": "100%",
                "width": "100%",
              },
            ]
          }
        />
      </View>
      <View
        accessibilityLabel="com.usereserva:id/category_card_item"
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "backgroundColor": "#ffffff",
            "borderColor": "#F8F8F8",
            "borderRadius": 8,
            "borderWidth": 1,
            "elevation": 0,
            "height": 150,
            "opacity": 1,
            "shadowColor": "#000000",
            "shadowOffset": {
              "height": 7,
              "width": 0,
            },
            "shadowOpacity": 0.6,
            "shadowRadius": 7,
            "width": 150,
          }
        }
        testID="com.usereserva:id/category_card_item"
      >
        <Image
          resizeMode="contain"
          source={
            {
              "uri": "https://images.ctfassets.net/6jsfqc13oxv4/7xrnXUOZChlV7WXHcyIOTC/595ecd21e49db1d98aef1fcab56c2766/Screenshot_from_2024-08-15_18-27-01.png",
            }
          }
          style={
            [
              {
                "height": 562.5,
                "width": 750,
              },
              {
                "height": "100%",
                "width": "100%",
              },
            ]
          }
        />
      </View>
      <View
        accessibilityLabel="com.usereserva:id/category_card_item"
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "backgroundColor": "#ffffff",
            "borderColor": "#F8F8F8",
            "borderRadius": 8,
            "borderWidth": 1,
            "elevation": 0,
            "height": 150,
            "opacity": 1,
            "shadowColor": "#000000",
            "shadowOffset": {
              "height": 7,
              "width": 0,
            },
            "shadowOpacity": 0.6,
            "shadowRadius": 7,
            "width": 150,
          }
        }
        testID="com.usereserva:id/category_card_item"
      >
        <Image
          resizeMode="contain"
          source={
            {
              "uri": "https://images.ctfassets.net/6jsfqc13oxv4/2PU3jmKTLkvNnfgFnhFqkM/e0e191ded28482be6a4270e6d2b0fd69/Screenshot_from_2024-08-15_18-27-35.png",
            }
          }
          style={
            [
              {
                "height": 562.5,
                "width": 750,
              },
              {
                "height": "100%",
                "width": "100%",
              },
            ]
          }
        />
      </View>
      <View
        accessibilityLabel="com.usereserva:id/category_card_item"
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "backgroundColor": "#ffffff",
            "borderColor": "#F8F8F8",
            "borderRadius": 8,
            "borderWidth": 1,
            "elevation": 0,
            "height": 150,
            "opacity": 1,
            "shadowColor": "#000000",
            "shadowOffset": {
              "height": 7,
              "width": 0,
            },
            "shadowOpacity": 0.6,
            "shadowRadius": 7,
            "width": 150,
          }
        }
        testID="com.usereserva:id/category_card_item"
      >
        <Image
          resizeMode="contain"
          source={
            {
              "uri": "https://images.ctfassets.net/6jsfqc13oxv4/Z2kpSFLXijhTdKQvgsG4j/85898ec1d0cde7a9951d418f556b684f/Screenshot_from_2024-08-15_18-27-55.png",
            }
          }
          style={
            [
              {
                "height": 562.5,
                "width": 750,
              },
              {
                "height": "100%",
                "width": "100%",
              },
            ]
          }
        />
      </View>
      <View
        accessibilityLabel="com.usereserva:id/category_card_item"
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "backgroundColor": "#ffffff",
            "borderColor": "#F8F8F8",
            "borderRadius": 8,
            "borderWidth": 1,
            "elevation": 0,
            "height": 150,
            "opacity": 1,
            "shadowColor": "#000000",
            "shadowOffset": {
              "height": 7,
              "width": 0,
            },
            "shadowOpacity": 0.6,
            "shadowRadius": 7,
            "width": 150,
          }
        }
        testID="com.usereserva:id/category_card_item"
      >
        <Image
          resizeMode="contain"
          source={
            {
              "uri": "https://images.ctfassets.net/6jsfqc13oxv4/1fkFj5WboCkITSdSeNqUiE/20aced11681c1512795bb59404a701dc/Screenshot_from_2024-08-15_18-29-04.png",
            }
          }
          style={
            [
              {
                "height": 562.5,
                "width": 750,
              },
              {
                "height": "100%",
                "width": "100%",
              },
            ]
          }
        />
      </View>
      <View
        accessibilityLabel="com.usereserva:id/category_card_item"
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "backgroundColor": "#ffffff",
            "borderColor": "#F8F8F8",
            "borderRadius": 8,
            "borderWidth": 1,
            "elevation": 0,
            "height": 150,
            "opacity": 1,
            "shadowColor": "#000000",
            "shadowOffset": {
              "height": 7,
              "width": 0,
            },
            "shadowOpacity": 0.6,
            "shadowRadius": 7,
            "width": 150,
          }
        }
        testID="com.usereserva:id/category_card_item"
      >
        <Image
          resizeMode="contain"
          source={
            {
              "uri": "https://images.ctfassets.net/6jsfqc13oxv4/5uIK3YxjuydYo4r6YPyXA1/ed24643656bc1b7b239eb6eddd476457/Screenshot_from_2024-08-15_18-28-49.png",
            }
          }
          style={
            [
              {
                "height": 562.5,
                "width": 750,
              },
              {
                "height": "100%",
                "width": "100%",
              },
            ]
          }
        />
      </View>
      <View
        accessibilityLabel="com.usereserva:id/category_card_item"
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "backgroundColor": "#ffffff",
            "borderColor": "#F8F8F8",
            "borderRadius": 8,
            "borderWidth": 1,
            "elevation": 0,
            "height": 150,
            "opacity": 1,
            "shadowColor": "#000000",
            "shadowOffset": {
              "height": 7,
              "width": 0,
            },
            "shadowOpacity": 0.6,
            "shadowRadius": 7,
            "width": 150,
          }
        }
        testID="com.usereserva:id/category_card_item"
      >
        <Image
          resizeMode="contain"
          source={
            {
              "uri": "https://images.ctfassets.net/6jsfqc13oxv4/nXPVcDj1RVY5NAbMe4OLQ/c6a182ef52aaa47c6d2d79c6bf70f9fa/Screenshot_from_2024-08-15_18-28-31.png",
            }
          }
          style={
            [
              {
                "height": 562.5,
                "width": 750,
              },
              {
                "height": "100%",
                "width": "100%",
              },
            ]
          }
        />
      </View>
      <View
        accessibilityLabel="com.usereserva:id/category_card_item"
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "backgroundColor": "#ffffff",
            "borderColor": "#F8F8F8",
            "borderRadius": 8,
            "borderWidth": 1,
            "elevation": 0,
            "height": 150,
            "opacity": 1,
            "shadowColor": "#000000",
            "shadowOffset": {
              "height": 7,
              "width": 0,
            },
            "shadowOpacity": 0.6,
            "shadowRadius": 7,
            "width": 150,
          }
        }
        testID="com.usereserva:id/category_card_item"
      >
        <Image
          resizeMode="contain"
          source={
            {
              "uri": "https://images.ctfassets.net/6jsfqc13oxv4/3SCuBDcJKbIABnbDdIBubK/861b6849f7836319d022e5038cf0981b/Screenshot_from_2024-08-15_18-28-16.png",
            }
          }
          style={
            [
              {
                "height": 562.5,
                "width": 750,
              },
              {
                "height": "100%",
                "width": "100%",
              },
            ]
          }
        />
      </View>
    </View>
  </View>
</View>
`;
